packages/ethereum-contracts/contracts/mocks/ERC777SenderRecipientMock.sol

```diff
@@ -170,6 +170,12 @@ contract ERC777SenderRecipientMock is Context, IERC777Sender, IERC777Recipient,
 contract ERC777RecipientReverting is IERC777Recipient, IERC1820Implementer {
```

Comment:
Noticed a minor thing from previous code already, in package.json

```
    "files": [
        "/contracts/**/*",
        "!/contracts/mocks/*",
        "/build/contracts/*.json",
        "!/build/contracts/*Mock*.json",
        "!/build/contracts/*Tester*.json",
        "/build/abi.js",
        "artifacts/**/*",
        "scripts/**/*",
        "test/TestEnvironment.js"
    ],
```

The npm packaging rule excludes any contract builds that has name Mock or Tester in it, that leaves these testing purpose contract being packaged:

```
$ grep -hR 'contract ' contracts/mocks/   | grep -Eiv 'mock|tester'
contract ERC777RecipientReverting is IERC777Recipient, IERC1820Implementer {
contract ERC777RecipientDrainingGas is IERC777Recipient, IERC1820Implementer {
contract ExclusiveInflowTestApp is SuperAppBase {
contract NonClosableOutflowTestApp is SuperAppBase {
contract SelfDeletingFlowTestApp is SuperAppBase {
contract ClosingOnUpdateFlowTestApp is SuperAppBase {
contract FlowExchangeTestApp is SuperAppBase {
    // using immutable, otherweise the proxy contract would see zero instead
// spliting this off because the contract is getting bigger
```

No big deal, can be dealt later.